app/views/waste_carriers_engine/registration_completed_forms/new.html.erb

```diff
@@ -4,7 +4,7 @@
 
     <p class="font-large"><%= t(".highlight_text") %><br>
 
-    <span class="strong"><%= @registration.reg_identifier %></span></p>
+    <span class="strong" id="reg-identifier"><%= @registration.reg_identifier %></span></p>
```

Comment:
Most IDs in page objects have underscore rather than hyphen. Could we use `reg_identifier` for consistency?